src/main/kotlin/com/team4099/robot2023/commands/manipulator/IntakeCubeCommand.kt

```diff
@@ -0,0 +1,29 @@
+package com.team4099.robot2022.commands.intake
+
+import com.team4099.robot2023.config.constants.ManipulatorConstants
+import com.team4099.robot2023.subsystems.manipulator.Manipulator
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.littletonrobotics.junction.Logger
+
+class IntakeCubeCommand(val manipulator: Manipulator) : CommandBase() {
```

Comment:
What you set the states to will depend heavily on where you are intaking from -- human player station cube intake will be a different manipulator state than ground intake, bc the manipulator position will likely be different

Reply:
building off this, it's worth splitting this command into these different states